### ISSUE TITLE
OrderModify failed with PendingOrders.

### ISSUE THREAD

**Issue by Perens28** [vdemydiuk/mtapi#117]
Hi,

I have issues with OrderModify when I want modify Pending orders. 

For example I want change SellStop on GOLD.

`apiclient.OrderModify(order.Ticket, 1300, 0, 0, DateTime.MaxValue);`

And I get error 130.  What I do wrong?

Thanks

**Comment by KptKuck** [comment]
Hello Perens,
look at this page and understud what is error 130.
[](url)https://book.mql4.com/trading/ordermodify

I think your new price is to close or against the order direction. 
Use RefreshRates bevore modify.

Change the price and see what happens

**Comment by Perens28** [comment]
In time when I try it, was GOLD about 1330. I try it manually as you can see in first topic and its return error. 
I solved it by delete order and make new one. But its not comfortable... 
Thanks for advice

**Comment by KptKuck** [comment]
Did you modify the correct order ?

**Comment by Perens28** [comment]
Yes I modify correct order. This working for you? Mofidy pending orders?
If I try modify SL or TP for BUY/SELL its ok. But pending orders not...

**Comment by KptKuck** [comment]
Post the complete Request structure for OrderSend .
like this
request.sl = 0;
request.tp = 0;
and so on

**Comment by KptKuck** [comment]
Iam not the devoloper of this API but i can help you to solve this problem.

Sorry i see that you use mtapi not mtapi5.

Enable debug mode in mtapi.mq4 like this
//#define __DEBUG_LOG__   old
#define __DEBUG_LOG__     new
compile and run 
see output and post ist after OrderModify Error

greets
